locale/en/blog/announcements/nodejs-certified-developer-program.md

```diff
@@ -0,0 +1,58 @@
++---
```

Comment:
Can you remove the `+` at the beginning of this line (as well as the next 7 lines). It should be in the format like:

```
---
title: ...
date: ...
category: ...
slug: ...
layout: blog-post.hbs
---
```